components/Cards/examples/CardExampleViewController.swift

```diff
@@ -45,6 +45,9 @@ class CardExampleViewController: UIViewController {
     cardScheme.colorScheme = colorScheme
     MDCCardThemer.applyScheme(cardScheme, to: card)
     card.isInteractable = false
+
+    imageView.isAccessibilityElement = true
+    imageView.accessibilityLabel = "Card Image"
```

Comment:
What is in the image? Reading this code (as a blind user would experience our interface) I cannot tell what the image contains.  If the image is just decorative, we can hide it from UIAccessibility.

Reply:
I added some meaningful content label, though that's a little out of context, since the rest of the card is just Lorem Ipsum...